packages/ember-glimmer/lib/helpers/-class.js

```diff
@@ -0,0 +1,25 @@
+import { InternalHelperReference } from '../utils/references';
+import { dasherize } from 'ember-runtime/system/string';
+
+function classHelper({ positional }) {
+  let path = positional.at(0);
+  let className = positional.at(1);
+  let value = path.value();
+
+  if (value === true) {
+    return dasherize(className.value());
```

Comment:
another way to think of it is the last arg is metadata, essential reflection of the path that was bound.  the first argument is the className binding, this is an internal helper refined from the `class={{boundProp}}` to add this additional metadata to handle the case of the boundProp not being a string but a boolean